hbase-balancer/src/main/java/org/apache/hadoop/hbase/favored/FavoredNodeAssignmentHelper.java

```diff
@@ -132,7 +132,7 @@ public static void updateMetaWithFavoredNodesInfo(
         puts.add(put);
       }
     }
-    MetaTableAccessor.putsToMetaTable(connection, puts);
+    MetaTableAccessor.putsToCatalogTable(connection, TableName.META_TABLE_NAME, puts);
```

Comment:
So, you keep referring to MetaTableAccessor so as to keep the patch small?

Reply:
Yes in places that don't include any real changes. This one is a mistake tho. It should've stuck with putsToMetaTable().